### PR TITLE
Exceptions handling

### LINKED ISSUE #78: Handle exceptions
When running the crawler on a specific website, if crawler encounters an exception, it stops to crawl this website and doesn't record anything in data files. However, I guess it could be better to continue working with this website after handling an exception if it is possible.

### PR DESCRIPTION
**Pull request by rhcu**
Fixes #78 
I was encountering "Element is not clickable at point <x, y> as it is obscured" exception, and as a solution, I found that we need to ensure that zoom is on 100%. And, I changed the place of the try-catch block in for loop in order to allow the crawler to finish crawling and writing to file if it encounters an exception.

### CODE REVIEW

File: crawler.py [hunk 1]
```diff
@@ -78,6 +82,8 @@ def do_something(driver):
     assert len(body) == 1
     body = body[0]
 
+    body.send_keys(Keys.CONTROL, 0)
```
Comment (marco-c) [inline crawler.py]: Why this change?

Comment (rhcu) [reply crawler.py]: `Element <a class="js-gps-track -link" href="/help"> is not clickable at point (135,655.7666625976562) because another element <div id="js-gdpr-consent-banner" class="p16 bg-black-700 fc-white ps-fixed b0 l0 r0 z-banner"> obscures it` - I was getting such kind of exceptions and Stackoverflow suggested that this can be an issue of incorrect (not 100%) zooming.

File: crawler.py [hunk 2]
```diff
@@ -163,19 +169,19 @@ def run(website, driver):
...
                 print('Can\'t find any element to interact with on {}'.format(website))
                 break
             saved_sequence.append(elem_attributes)
 
             print('  - Using {}'.format(elem_attributes))
-    except TimeoutException as e:
-        # Ignore timeouts, as they are too frequent.
-        traceback.print_exc()
-        print('Continuing...')
+        except (TimeoutException, ElementNotInteractableException, StaleElementReferenceException, WebDriverException):
```
Comment (marco-c) [inline crawler.py]: Can you post examples of these exceptions? Maybe it's fine to ignore `ElementNotInteractableException` and `StaleElementReferenceException`, but maybe we shouldn't ignore `WebDriverException`.

File: crawler.py (hunk 1 above)
Comment (marco-c) [reply crawler.py]: Was the issue fixed with this?

Comment (rhcu) [reply crawler.py]: Not fully but it decreased the number of occurrences.

Comment (marco-c) [reply crawler.py]: OK!

File: crawler.py (hunk 2 above)
Comment (rhcu) [reply crawler.py]: `StaleElementReferenceException` - selenium.common.exceptions.StaleElementReferenceException: Message: The element reference of a class="select-menu-item js-navigation-item js-navigation-open " href="/mozilla/bedrock/blob/bug1150328-wom-landing-page/bedrock/mozorg/templates/mozorg/home/home-en.html" is stale; either the element is no longer attached to the DOM, it is not in the current frame context, or the document has been refreshed
`ElementNotInteractableException` - selenium.common.exceptions.ElementNotInteractableException: Message: Element \<a href="/users/496127/pwan"\> could not be scrolled into view
`WebDriverException` - selenium.common.exceptions.WebDriverException: Message: Element <button class="ProfileTweet-actionButton js-actionButton js-actionFavorite" type="button"> is not clickable at point (637,9.350006103515625) because another element <div class="container"> obscures it

Comment (rhcu) [reply crawler.py]: Should I remove `WebDriverException`? It is quite frequent and interrupts crawling. Can you suggest another workaround to fix this?

Comment (marco-c) [reply crawler.py]: Let's keep it, I didn't know what kinds of exceptions would be hidden behind WebDriverException, but the one you mentioned seems not too bad.
